tests/init_tests.py

```diff
@@ -0,0 +1,11 @@
+def test_initialize_function():
+    nx = 100
+    L = 1
```

Comment:
L should be written as 1.0 because the length of the domain (L) represents a continuous physical quantity which typically involves real (floating-point) numbers, not just integers.